components/perf-tools/likwid/SPECS/likwid.spec

```diff
@@ -85,6 +86,12 @@ It consists of:
           COMPILER="GCC" \
           FC="gfortran" \
           FCFLAGS="-J ./ -fsyntax-only" \
+%endif
+%endif
+%ifarch aarch64
+          COMPILER="GCCARMv8" \
```

Comment:
Which dependency provides `GCCARMv8` ?

Reply:
I noticed the `For ARM builds, the COMPILER flag in config.mk needs to changed to GCCARMv8 or ARMCLANG (experimental). ` at https://github.com/RRZE-HPC/likwid#download-build-and-install

Reply:
`GCCARMv8` is the LIKWID-specific build option for ARM8 architectures with `gcc` as compiler. `ARMCLANG` uses the `armclang` compiler.